labgrid/remote/exporter.py

```diff
@@ -669,6 +669,26 @@ def _get_params(self):
 
 exports["LXAIOBusPIO"] = LXAIOBusNodeExport
 
+@attr.s(eq=False)
+class YKUSHPowerPortExport(ResourceExport):
+    """ResourceExport for YKUSHPowerPort devices"""
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        local_cls_name = self.cls
+        self.data['cls'] = f"{local_cls_name}"
+        from ..resource import ykushpowerport
+        local_cls = getattr(ykushpowerport, local_cls_name)
+        self.local = local_cls(target=None, name=None, host=self.host, **self.local_params)
```

Comment:
I don't know if that's needed or whether it's correct or not. @Emantor, @jluebbe: could you comment on this?